### PR TITLE
align all ls command options

### PR DESCRIPTION
**Pull request by zubenkoivan**

### CODE REVIEW

File: neuro-cli/src/neuro_cli/image.py [hunk 1]
```diff
@@ -137,9 +149,21 @@ async def ls(
     with root.status("Fetching images"):
         images = await root.client.images.list(cluster_name=cluster)
 
-    if project:
-        projects = set(project)
-        images = [image for image in images if image.project_name in projects]
+    if all_orgs:
```
Comment (YevheniiSemendiak) [inline neuro-cli/src/neuro_cli/image.py:152]: Our image name parser works incorrect - it threats org names as project names and the underlying project name as a part of image name.
AFAIU, this affects only the representation (and filtering here), so we could fix it later / in a separate PR.

example:
(current config: testproj + no-org, testorg is an org with two projects, yevheniisemendiak and testproj and tp4 is a project w.o. org the current user has access to)
image

File: neuro-sdk/src/neuro_sdk/_jobs.py [hunk 2]
```diff
@@ -518,6 +519,8 @@ async def list(
             params.add("being_dropped", str(_being_dropped))
         if _logs_removed is not None:
             params.add("logs_removed", str(_logs_removed))
+        for org_name in org_names:
+            params.add("org_name", org_name or "NO_ORG")
```
Comment (YevheniiSemendiak) [inline neuro-sdk/src/neuro_sdk/_jobs.py:523]: As I could see, we do not support `NO_ORG` filter on the API side yet.

**Review by YevheniiSemendiak — COMMENTED** [review]
Great job!
Some suggestions here. We might address most of them in a separate PRs, but I would rather suggest resolving default filtering of `neuro job ps` (which is broken atm).

File: neuro-sdk/src/neuro_sdk/_jobs.py (hunk 2 above)
Comment (YevheniiSemendiak) [reply neuro-sdk/src/neuro_sdk/_jobs.py:523]: submited https://github.com/neuro-inc/platform-api/issues/2096

File: neuro-cli/src/neuro_cli/secrets.py [hunk 3]
```diff
@@ -31,44 +32,57 @@ def secret() -> None:
...
         )
 
+    if all_orgs:
+        org_name = ORG_NAME_SENTINEL
+    else:
+        org_name = parse_org_name(org, root)  # type: ignore
+
+    if all_projects:
+        project_name = None
+    else:
```
[the remark this answers is not in the document]
Comment (zubenkoivan) [reply neuro-cli/src/neuro_cli/secrets.py:79]: I doubt that they are conflicting. They just select orgs/projects set to show, and `--all-orgs` covers `--org` option.

File: neuro-sdk/src/neuro_sdk/_jobs.py (hunk 2 above)
Comment (zubenkoivan) [reply neuro-sdk/src/neuro_sdk/_jobs.py:523]: thanks, have already started working on it

File: neuro-cli/src/neuro_cli/image.py (hunk 1 above)
Comment (zubenkoivan) [reply neuro-cli/src/neuro_cli/image.py:152]: I will do it in a separate pr. AFAIU it was broken when we introduced orgs, parser doesn't recognize non current orgs and treats them as project.

File: neuro-cli/src/neuro_cli/secrets.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (YevheniiSemendiak) [reply neuro-cli/src/neuro_cli/secrets.py:79]: True, the code works as expected, I just refer to UX. This is minor

**Review by YevheniiSemendiak — APPROVED** [review]
LGTM!

**Pull request merged**